dll/win32/shell32/CDefView.cpp

```diff
@@ -545,6 +545,9 @@ void CDefView::UpdateStatusbar()
     WCHAR szPartText[MAX_PATH] = {0};
     UINT cSelectedItems;
 
+    if (!m_ListView)
```

Comment:
I think that this check can stay, however.

Reply:
OK.